src/tests/functional/plugin/cpu/single_layer_tests/gru_cell.cpp

```diff
@@ -2,38 +2,64 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-#include "ngraph/op/gru_cell.hpp"
-#include <shared_test_classes/single_layer/gru_cell.hpp>
+#include "shared_test_classes/base/ov_subgraph.hpp"
```

Comment:
for all tests please
+ add cases whet non weight input fully undefined [?, ?, ?]
+ separate instance on smoke and nightly

Reply:
Done.